chromadb/api/models/Collection.py

```diff
@@ -311,7 +317,7 @@ def delete(
         ids: Optional[IDs] = None,
         where: Optional[Where] = None,
         where_document: Optional[WhereDocument] = None,
-    ) -> List[str]:
+    ) -> None:
```

Comment:
The underlying delete does not return the ids, it returns UUIDs which are meaningless to the client. We should follow the docstring and return None.